bluebutton-data-pipeline-benchmarks/src/test/ansible/benchmark_etl.yml

```diff
@@ -20,6 +20,34 @@
     - lineinfile: regexp="^Defaults    requiretty" dest=/etc/sudoers state=absent
       become: true
 
+- name: Create FHIR DB
+  hosts: etl
+  user: "{{ ssh_user }}"
+  gather_facts: true
+
+  tasks:
+
+    - name: Install Pre-requisites
+      yum:
+        name: "{{ item }}"
+        state: latest
+      with_items:
+        # Needed to use the `postgresql_db` module.
+        - python-psycopg2
+      become: true
+    
+    - name: Create FHIR DB
+      postgresql_db:
+        login_host: "{{ hostvars['rds']['endpoint'] }}"
+        port: "{{ hostvars['rds']['port'] }}"
+        login_user: "{{ postgres_master_username }}"
+        login_password: "{{ postgres_master_password }}"
+        name: fhirdb
```

Comment:
Perhaps, yes, but this particular deployment script is only used in the benchmarks:

1. This playbook (and the `provision.yml` one) is run multiple times, with different values for `iteration_index` from `0..S3ToDatabaseLoadAppBenchmark.NUMBER_OF_ITERATIONS`.
1. Each set of executions stands up a new set of systems to benchmark against (in `provision.yml`) and then runs the benchmarks against them (in `benchmark_etl.yml`).
1. Once the benchmarks are completed, the `teardown.yml` script is run for each set of benchmark systems to scrape the results off of them and shut them down.

This code right here doesn't affect the AWS dev/test/prod environments.

What I'm doing with this pull request, though, is ensuring that this whole complicated benchmark setup shares some code with the other Ansible playbooks that also setup an ETL server. Specifically, the [karlmdavis.bluebutton_data_pipeline Ansible role](https://github.com/HHSIDEAlab/bluebutton-ansible-role-data-pipeline) is now used by all three of these Ansible playbooks:

* These benchmarks, which are part of [bluebutton-data-pipeline](https://github.com/HHSIDEAlab/bluebutton-data-pipeline).
* The [bluebutton-ansible-playbooks-backend](https://github.com/HHSIDEAlab/bluebutton-ansible-playbooks-backend) playbooks for the AWS dev/test/prod environments.
* The [bluebutton-sandbox](https://github.com/HHSIDEAlab/bluebutton-sandbox) playbooks for the HHS IDEA Lab developer sandbox.

Any part of that that I could explain better?